src/components/dialog/examples/dialog-closing-actions.tsx

```diff
@@ -40,15 +34,15 @@ export class DialogClosingActionsExample {
         ];
     }
 
-    private triggerOnClick() {
+    private triggerOnClick = () => {
```

Comment:
❤️

I've been thinking for a while I should go through and make these changes. Thank you! 🙏

Reply:
I'm not sure about the commit message though? "improve ui for config options in examples"?

Am I just being daft again? 😆

Reply:
😄  I don't know dude. I just copied one of your own commit messages for an exact change in another component 😅 (don't remember  where it was, but I guess it was when we fixed `readonly` styles for the input field.

Reply:
but feel free to change. I have no clue what "improve ui for config options in examples" means 😄

Reply:
Ah! Ok, I found the commit. The change that I was referring to there wasn't the change from `function` to `=>`, it was the fact that I had changed from using limel-switch and limel-button in a few different examples, to using limel-checkbox, for consistency…

I should have split that commit into two separate ones I guess 😄

Reply:
I'll come up with some better commit message. Tomorrow. It's time to log off for today 😅

Reply:
⚡